src/Resolve/graphtype.jl

```diff
@@ -400,7 +400,7 @@ function _add_reqs!(graph::Graph, reqs::Requires, reason)
     pvers = graph.data.pvers
 
     for (rp,rvs) in reqs
-        haskey(pdict, rp) || error("unknown required package $(pkgID(rp, graph))")
+        haskey(pdict, rp) || error("unknown required package $(logstr_pkgID(rp, graph))")
```

Comment:
This isn't a log message this is an error so probably should not change it 
```suggestion
        haskey(pdict, rp) || error("unknown required package $(pkgID(rp, graph))")
```